README.md

```diff
@@ -15,7 +15,7 @@ The WPILib Mission is to enable FIRST teams to focus on writing game-specific so
 
 # Getting SmartDashboard
 
-The latest release build of SmartDashboard can be found on [WPI's server](http://first.wpi.edu/FRC/roborio/maven/release/edu/wpi/first/wpilib/SmartDashboard/).
+The latest release build of SmartDashboard can be found on [WPI's server](http://first.wpi.edu/FRC/roborio/maven/release/edu/wpi/first/wpilib/SmartDashboard/).  The largest file listed is the jar file.  You can check by hovering over the link which will reveal the full path, including the extension.
```

Comment:
When you click on this link, you still have to go into the version folder. You should add this step in (or edit the server to have a `latest` folder and change the link to this folder)